msg/en_US/csql.msg

```diff
@@ -188,6 +191,13 @@ $ csql help messages
    ;HISTORYRead <history_num>   - read entry on the history number into command buffer.\n\
    ;TRAce [ON/OFF] [text/json]  - enable/disable sql auto trace.\n\
    ;SIngleline [ON|OFF]         - enable/disable single-line mode.\n\
+   ;Connect [user-name] [database-name]@[host-name]\n\
+                                - connect user, database, host.\n\ 
```

Comment:
I suggest as below.
- connect to database 
- csql session connection

Reply:
I am a little bit concerned. because switch does not seem so intuitive.

`12. to shift, change, turn aside, or change the direction of (something)
13. to exchange (places); replace(something by something else)` - Collins dictionary.

 ;Connect disconnects and then connects so  I think exchange or replace meaning is a little bit far.

Reply:
I think
```
   ;Connect username [dbname][@hostname]\n\
                                - connect user database@host.\n\
```

Reply:
@kisoo-han @airnet73 
I checked .msg files but it is usually not unified.
Sometimes database-name and dbname or database-name@hostname and database-name[@host]

It makes me confused.

Reply:
I recommend the kisoo-han opinion.

Reply:
I see. I will apply it.